app/assets/javascripts/search_with_map.js

```diff
@@ -144,7 +146,8 @@ navigatorController.addMarkers = function () {
 navigatorController.getTasks = function (taskees) {
   var tasks = []
   taskees.map(function (taskee) {
-    if (taskee.distance <= 50) tasks.push(taskee.tasks)
+    if (taskee.distance <= 50)
+    tasks.push(taskee.tasks)
```

Comment:
**Critical - Code Quality**
Braces are missing and also, please indent properly.